curvefs/proto/metaserver.proto

```diff
@@ -196,6 +198,7 @@ message Inode {
     map<uint64, S3ChunkInfoList> s3ChunkInfoMap = 18; // TYPE_S3 only, first is chunk index
     optional uint32 dtime = 19;
     optional uint32 openmpcount = 20; // openmpcount mount points had the file open
```

Comment:
> repalce inode with { attr xattr volumeExtentList s3ChunkInfoMap }

I tried this form at first and it will change all the files exists. If need, can fix in another pr later.

Reply:
add a todo here

Reply:
> add a todo here

fixed